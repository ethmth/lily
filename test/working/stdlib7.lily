
let a::int = [3, 2, 1]

printl(a)
let res:int = pop(a)

printl(a)
print(res)

res = pop(a)

printl(a)
print(res)

a.append(5)

printl(a)

res = pop(a)

printl(a)
print(res)


