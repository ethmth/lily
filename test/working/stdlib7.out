[3, 2, 1]
[3, 2]
1
[3]
2
[3, 5]
[3]
5
